web/angular-wallet/README.md

```diff
@@ -23,3 +23,28 @@ Run `ng test` or `npm test` to execute the unit tests via [Jest](https://jestjs.
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+
+## Releasing
+
+- Electron: Tag the release (e.g. desktop-1.0.0) and push it up. Travis will build and deploy the electron app. 
```

Comment:
Important to point out the correct tag format, i.e needs to start with `desktop-<semver>`, otherwise no build is being triggered.